blocks/scaip-sidebar.php

```diff
@@ -17,6 +17,10 @@ function scaip_sidebar_block_init() {
 		return false;
 	}
 
+	if ( apply_filters( 'scaip_disable_sidebars', false ) ) {
```

Comment:
> Repeat the steps from "block-based widgets" and confirm it also works as expected

When I visit the blocks-based Widgets editor while not in legacy mode, I'm unable to add the "Inserted Ad Position" block to any widget area. Digging in a bit, it looks like the value of `	if ( apply_filters( 'scaip_disable_sidebars', false )` is `true`, so the block is never getting registered.